lectures/time_series_models/additive_functionals.md

```diff
@@ -335,14 +335,14 @@ function multiplicative_decomp(A, B, D, F, ν, nx)
 end
```

Comment:
i think additivve_decomp and multiplicative_decomp can be combines into one `decomp` function - lines that call for either function should be edited too